api/internal/git/repospec.go

```diff
@@ -30,16 +30,16 @@ type RepoSpec struct {
 	// Host, e.g. https://github.com/
 	Host string
 
-	// orgRepo name (organization/repoName),
+	// RepoPath name (organization/repoName),
```

Comment:
nit: Can we change the doc string for `RepoPath` to be more descriptive? For instance, 

`RepoPath` is the relative path to the repository on `Host`. 

Right now it still sounds too specific to `OrgRepo`.